example/src/WebView.tsx

```diff
@@ -36,15 +52,26 @@ const WebView: React.FC<Props> = ({ navigation, ...props }) => {
     [navigation]
   );
 
+  const onWebViewMount = useCallback(() => {
+    setWebViewVisibilityState(true);
+  }, [setWebViewVisibilityState]);
+
+  const onWebViewUnmount = useCallback(() => {
+    setWebViewVisibilityState(false);
+  }, [setWebViewVisibilityState]);
+
```

Comment:
Previous code in `useFocusEffect` was being executed to early - `sessionHandle` prop (at least on iOS) was not initialised and caused crash. Introducing props which make sure that native side is properly mounted might be a better way to go.